src/com/videojs/providers/HLSProvider.as

```diff
@@ -277,6 +278,37 @@ package com.videojs.providers{
           }
         }
 
+       /**
+         * Should return the list of audio-tracks that this content has.
+         */
+        public function get audioTracks():Array {
+            var _audioTracks:Array = [];
+
+            for each(var _audioTrackId:AudioTrack in _hls['audioTracks']) {
```

Comment:
maybe name this `_audioTrack` instead of `_audioTrackId` since its a full audio track object and not just the id